Extension/translations_auto_pr.js

```diff
@@ -115,14 +113,21 @@ if (!hasAnyChanges()) {
     return;
 }
 
-// Commit changes files.
-console.log(`Commiting changes (git commit -m "${commitComment}")`);
-cp.execSync(`git commit -m "${commitComment}"`);
+// Set up user and permissions(Never run this locally)
```

Comment:
What happens if it's run locally? Should there be some guard against that in case?

Reply:
I set the values using script args, so if run locally the right local values could be passed in.  I had to set them as there is an error if unset on commit, on the build vm.  I'm not sure how to guard against it.  I could do so more digging to find a way.